scripts/azure-linux_mac.yml

```diff
@@ -169,14 +175,16 @@ steps:
       export ASAN_OPTIONS=detect_leaks=0 LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libasan.so.4
     fi
 
-    if [[ "$TILEDB_GCS" == "ON" ]]; then
-      # GCS unit tests are temporarily unsupported on CI. Fake success with
+    if [[ "$AGENT_OS" == "Darwin" && "$TILEDB_GCS" == "ON" ]]; then
+      # GCS unit tests are temporarily unsupported on CI for MACOS. Fake success with
       # this echo.
       echo "##vso[task.setvariable variable=TILEDB_CI_SUCCESS]1"
     else
-      make check	
+      make check
     fi
 
+    make check
```

Comment:
I think this was left by mistake when re-adding the logic on L178 from an earlier commit. Remove this line.